homeassistant/components/wiz/manifest.json

```
@@ -0,0 +1,14 @@
+{
+  "domain": "wiz",
+  "name": "WiZ Platform",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/wiz_light",
+  "issue_tracker": "https://github.com/sbidy/wiz_light/issues",
```

Issue tracker not needed. Docs still using old domain.

```suggestion
  "documentation": "https://www.home-assistant.io/integrations/wiz",
```